problem2.R

```diff
@@ -0,0 +1,109 @@
+
+# Skeleton file 2 for Assignment 1 in BAN400. 
+# -------------------------------------------
+
+# More detailed steps to complete Problem 1.
+
+library(tidyverse)    # Contains most of what we need.
+
+# Read the entire data file into memory using the readLines()-function. Use the
+# URL direcly or read the data from the local file that is in the repository.
+
+# The readLines()-function puts each line into a separate element in a character
```

Comment:
Use your own comments in the final submission. These comments are meant to be a guide for you and not your own comments.